src/Fixer/Import/GroupImportFixer.php

```diff
@@ -89,7 +89,7 @@ private function getSameNamespaces(Tokens $tokens): array
             $namespaceA = $this->getNamespaceNameWithSlash($a);
             $namespaceB = $this->getNamespaceNameWithSlash($b);
 
-            $namespaceDifference = \strlen($namespaceA) - \strlen($namespaceB);
+            $namespaceDifference = \strlen($namespaceA) <=> \strlen($namespaceB);
 
             return 0 !== $namespaceDifference ? $namespaceDifference : strcmp($a->getFullName(), $b->getFullName());
```

Comment:
```suggestion
            return 0 !== $namespaceDifference
                ? $namespaceDifference
                : $a->getFullName() <=> $b->getFullName();
```

Reply:
hm... i think we could replace most `strcmp` with `<=>` ;)